### PR TITLE
New fields for the DFRN protocol (XMPP and Account Type)

### PR DESCRIPTION
**Pull request by annando**
There are now two new fields that are transmitted via DFRN.

The first is the XMPP address. It can be entered with the profile editor. The value will be shown on the contact page and at the contact posts in the "display" page.

The second is a new field: "Account Type". This should lead to lesser confusion about the page type so that people doesn't mix community forums with personal accounts, ...

This field isn't read by now (this will follow with another pull request). In the future we could even search for news pages (to make it easier for users to find news reflectors). Additionally we could hide some profile data, depending on the type.

### CODE REVIEW

**Comment by annando** [comment]
According to the discussion with @rabuzarus I will rename the "company" to "organization" and will add some additional checks.

**Comment by annando** [comment]
The changes are done. Now "company" is named "organisation" (which is much more neutral) and there is an update routine that changes the account type to "community" if the page flag tells us so. Additionally there is a similar routine in the settings as some kind of double check.

File: boot.php [hunk 1]
```diff
@@ -182,6 +182,17 @@
 /** @}*/
 
 /**
+ * @name account types
+ *
```
Comment (annando) [inline boot.php]: Done

**Comment by annando** [comment]
I hope that now everyone is lucky? :-)

**Comment by rabuzarus** [comment]
I am ;-)
@tobiasd ????

**Comment by tobiasd** [comment]
See that other thread ;-)

**Pull request merged**